otherfile/version_check.py

```diff
@@ -29,11 +29,13 @@
 PARAMS_ITEMS = ['ART_VERSION = "{0}"']
 META_ITEMS = ['% set version = "{0}" %']
 ISSUE_TEMPLATE_ITEMS = ["- Art {0}"]
+SECURITY_ITEMS = ["| {0}           | :white_check_mark: |", "| < {0}         | :x:                |"]
 
 FILES = {
     "setup.py": SETUP_ITEMS,
     "INSTALL.md": INSTALL_ITEMS,
     "CHANGELOG.md": CHANGELOG_ITEMS,
+    "SECURITY.md": SECURITY_ITEMS,
```

Comment:
I think we don't need to update this each time.
I know that we are using the scikit-learn template but it's wired to update the security-supported version each time after a release.

Reply:
Here we have two options:

1. Support only one version (the latest stable one)
2. Support a range of versions

* In the second option if someone finds a vulnerability in a supported version, which is not the latest, we should release a specific bugfix for that version. You can refer to how different Python versions are maintained.

Reply:
If you disagree with the correct option, we can discuss it further.